cli/cli/command_str_consts/command_str_consts.go

```diff
@@ -57,5 +57,7 @@ const (
 	PathCmdStr              = "path"
 	VersionCmdStr           = "version"
 	GatewayCmdStr           = "gateway"
-	LspCmdStr               = "lsp"
 )
+
+// TODO: added constant error message here, can we move to another file later.
+const ErrorMessageDueToStarlarkFailure = "Kurtosis execution threw an error. See output above for more details"
```

Comment:
you could use an error type here 
`var ErrorMessageDueToStarlarkFailure   = errors.New("Kurtosis execution threw an error. See output above for more details")`